Doc/library/re.rst

```diff
@@ -1588,8 +1595,9 @@ Note however that in :const:`MULTILINE` mode :func:`match` only matches at the
 beginning of the string, whereas using :func:`search` with a regular expression
 beginning with ``'^'`` will match at the beginning of each line. ::
 
-   >>> re.match('X', 'A\nB\nX', re.MULTILINE)  # No match
-   >>> re.search('^X', 'A\nB\nX', re.MULTILINE)  # Match
+   >>> re.match("X", "A\nB\nX", re.MULTILINE)  # No match
+   >>> re.fullmatch("X", "A\nB\nX", re.MULTILINE)  # No Match
```

Comment:
And delete this`fullmatch` line also. This section doesn't need to mention fullmatch.